addons/binding/org.openhab.binding.ambientweather/src/main/java/org/openhab/binding/ambientweather/internal/json/EventDataJson.java

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ambientweather.internal.json;
+
+/**
+ * The {@link EventDataJson} is the JSON object returned
+ * from the Ambient Weather real-time API for the weather stations
+ * whose weather data is hosted on ambientweather.net. This object is
+ * generated whenever the data for the station changes on
+ * ambientweather.net.
+ *
+ * @author Mark Hilbush - Initial Contribution
+ */
+public class EventDataJson {
```

Comment:
Can you please rename the json package to "model" or "dto"? That is the established name for data transfer objects.

Reply:
Done